fixit/cli/apply_fix.py

```diff
@@ -144,6 +144,9 @@ def apply_fix_operation(
                         get_lint_config().formatter, updated_source
                     )
                 with open(path, "wb") as f:
+                    # pyre-fixme[6]: Expected `Union[array.array[typing.Any],
+                    #  bytearray, bytes, ctypes._CData, memoryview, mmap.mmap]` for 1st
+                    #  param but got `str`.
```

Comment:
Oh cool, this (and the one below) are genuine bugs :) If `updated_source` has some non-ASCII characters, Fixit will clobber the file with invalid encoding.

We should open the file without binary mode:
```suggestion
                with open(path, "w") as f:
```